tests/basic_hybrid.py

```diff
@@ -0,0 +1,67 @@
+"""
+  This file is a smoke test for the CLI parsing.
+  Commands are executed as they are executed from a command line.
+"""
+
+from __future__ import print_function
+
+import os
+
+import pytest
+
+from digger import parser
+from digger.helpers import android as android_helper
+
+
+current_dir = os.path.dirname(os.path.realpath(__file__))
+tmp_dir = '%s/tmp' % current_dir
+app_dir = '/home/lrossetti/Containers/basic'
```

Comment:
actually I forgot to remove this file, the conftest.py file already downloads a cordova light build